guides/sev.md

```diff
@@ -1,18 +1,113 @@
+## Getting Started
 
-[AMD SEV - Prepare Host OS](https://github.com/AMDESE/AMDSEV#prepare-host-os)
+This guide covers platform-specific setup for SEV and walks through the complete flows for the different CoCo use cases:
+
+- [Container Launch with Memory Encryption](#container-launch-with-memory-encryption)
+- [Pre-Attestation Utilizing Signed and Encrypted Images](#pre-attestation-utilizing-signed-and-encrypted-images)
+
+## Container Launch With Memory Encryption
+
+### Launch a Confidential Service 
+
+To launch a container with SEV memory encryption, the SEV runtime class (`kata-qemu-sev`) must be specified as an annotation in the yaml. A base alpine docker container ([Dockerfile](https://github.com/kata-containers/kata-containers/blob/main/tests/integration/kubernetes/runtimeclass_workloads/confidential/unencrypted/Dockerfile)) has been previously built for testing purposes. This image has also been prepared with SSH access and provisioned with a [SSH public key](https://github.com/kata-containers/kata-containers/blob/main/tests/integration/kubernetes/runtimeclass_workloads/confidential/unencrypted/ssh/unencrypted.pub) for validation purposes.
+
+Here is a sample service yaml specifying the SEV runtime class: 
+
+```yaml
+kind: Service
+apiVersion: v1
+metadata:
+  name: "confidential-unencrypted"
+spec:
+  selector:
+    app: "confidential-unencrypted"
+  ports:
+  - port: 22
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: "confidential-unencrypted"
+spec:
+  selector:
+    matchLabels:
+      app: "confidential-unencrypted"
+  template:
+    metadata:
+      labels:
+        app: "confidential-unencrypted"
+       annotations:
+        io.containerd.cri.runtime-handler: kata-qemu-sev
+    spec:
+      runtimeClassName: kata-qemu-sev
+      containers:
+      - name: "confidential-unencrypted"
+        image: ghcr.io/kata-containers/test-images:unencrypted-nightly
+        imagePullPolicy: Always
+```
+
+Save the contents of this yaml to a file called `confidential-unencrypted.yaml`.
+
+Start the service:
+
+```shell 
+kubectl apply -f confidential-unencrypted.yaml
+```
+
+Check for errors:
+
+```shell
+kubectl describe pod confidential-unencrypted
+```
+
+If there are no errors in the Events section, then the container has been successfully created with SEV memory encryption.
+
+### Validate SEV Memory Encryption
+
+The container dmesg log can be parsed to indicate that SEV memory encryption is enabled and active. The container image defined in the yaml sample above was built with a predefined key that is authorized for SSH access.
+
+Get the pod IP:
+
+```shell
+pod_ip=$(kubectl get pod -o wide | grep confidential-unencrypted | awk '{print $6;}')
+```
+
+Download and save the SSH private key and set the permissions.
+
+```shell
+wget https://github.com/kata-containers/kata-containers/raw/main/tests/integration/kubernetes/runtimeclass_workloads/confidential/unencrypted/ssh/unencrypted -O confidential-image-ssh-key
```

Comment:
this could be a permanent link to keep it working (OTOH, we have the link checker that notices if the key disappears)

Reply:
Hi Mikko, I've made a change in the line before this point. Is that what you had in mind? Or were you thinking of something else?

Reply:
my thinking was to use a permanent link, such as, https://raw.githubusercontent.com/kata-containers/kata-containers/6a9266124bbde0b29c39df5a9ff74cca8a35c04d/tests/integration/kubernetes/runtimeclass_workloads/confidential/unencrypted/ssh/unencrypted but I'm OK as it is now also (since we should see any link breaks)

Reply:
@mythi Ok, do you think this pr is good to be merged then?